Modules/Invoice/Config/config.php

```diff
@@ -15,5 +15,9 @@
             'email' => env('INVOICE_UNPAID_LIST_EMAIL', 'finance@coloredcow.com'),
             'name' => env('INVOICE_UNPAID_LIST_NAME', 'ColoredCow Finance')
         ]
-    ]
+    ],
+
+    'indian' => 'indian',
+    'international' => 'international',
```

Comment:
You should put these 2 values inside another array that can define what these values are for.
```suggestion
    'region' => [    
        'indian' => 'indian',
        'international' => 'international',
    ]
```

Reply:
Changed everywhere